boards/arm/mimxrt1010_evk/mimxrt1010_evk.dts

```diff
@@ -82,10 +82,11 @@ arduino_serial: &lpuart1 {};
 };
 
 &flexspi {
+	status = "okay";
 	reg = <0x400a0000 0x4000>, <0x60000000 DT_SIZE_M(16)>;
 	at25sf128a: at25sf128a@0 {
 		compatible = "nxp,imx-flexspi-nor";
-		size = <134217728>;
```

Comment:
The flash capacity on this board is 128Mb

Reply:
I just checked the calculation and 1024*1024*16*8 is 134217728

Reply:
Yeah I checked the number before/after myself just in case, flash size here is in bit while above is in bytes, few boards are using the `DT_SIZE_M(16 * 8)` style so it's easier to match with the one above and less likely to miss changing one of the two when copying it to a custom board with a different flash size.